02-Custom-Login/app/controllers/home_controller.rb

```diff
@@ -0,0 +1,35 @@
+require "auth0"
+
+class HomeController < ApplicationController
+  attr_reader :client
+
+  def auth
+    v2_creds = { client_id: ENV['AUTH0_CLIENT_ID'],
+     token: '',
```

Comment:
We don't need a token to login with user and password and its mandatory for v2 credentials